### ISSUE TITLE
New bot commands, part1 

### ISSUE THREAD

**Issue by vividkp** [natzert/PrintMakingBot#22]
Hey!  Here are some new commands the mods think will be useful.  There are some others that need a bit of Dave input, but we can start with these.   Thanks! 

!faces | The pictures you see on the lampposts are old Japanese comedians who became famous while working in the theaters that were located along this street.
!streetcam | The camera in the lower left corner shows the street in front of the shop. Sometimes it points east, toward Senso-ji Buddhist temple and Tokyo Skytree, and sometimes it points west, toward the Don Quijote store down the street.
!japan-since | Dave has lived in Japan since 1986.
!japanese | Dave speaks workable Japanese.
!sharpen | Dave sharpens all of his tools himself almost daily. Sometimes he sharpens his knife on the stream.
!print-stream | Printing is a little harder to do on-stream than carving, because timing is really important. The paper and block need to be at the optimal moisture content to achieve a good impression, and it's too easy while streaming to get distracted and let something dry out too much. That said, it does happen sometimes!
!depth | Dave determines the depth to carve based on the distance between lines. If there are small gaps, the depth doesn't need to be deep, but if there are large gaps, he will carve a depth up to 4mm. This is to make sure the paper doesn't touch the wrong part of the wood and get pigment where it should not.
!twitch-since | Dave has been streaming on Twitch since August 2017.
!embossing | As the last step before preparing the prints for shipment, Dave embosses the names of the carver and printer into the white border. To do this he uses a polymer plate carrying the correct names combination that he orders from a regular print-shop. He moistens this area slightly for the embossing to work.
!origin | You can learn more about Dave's origin story in this video: https://www.youtube.com/watch?v=2L3M7isGQbg
!myst | Dave used to play Myst with his daughters, so Jed (designer of many of the prints Dave makes) sent him the shirt as a gift a few years ago.

**Comment by vividkp** [comment]
Here are a few more that Dave has edited: 

!interns | Dave doesn't offer internships/training, but occasionally he will allow someone who is already producing woodblock prints to work for a while in the Mokuhankan workshop.

!imperfect | Mokuhankan does not sell or give away the prints that end up with defects. Neither the designer, the craftspeople, nor Dave himself wants those prints out in the world. This is the same as with any manufacturing business - a certain percentage of the products simply don't make it through all the way to the end. Don't lose any sleep over this, please.

!scope | The scope is a Meiji EMZ5 binocular microscope, like those used to examine circuit boards. Dave uses an accessory lens that reduces the magnification by 0.5X (which also increases the field of view). The main effect is to bring the work closer to the eyes without having to slump over. Dave started using the scope while he was producing his 'Beauties of Four Season' series, to help with the fine hair carving, and hasn't looked back! (Not to mention that fact that in recent years his eyesight has begun to fail, and having the scope is going to extend his working career at least a few years, it seems.)

**Comment by natzert** [comment]
Hey @vividkp. One question, you have this:

> !depth | Dave determines the depth to carve based on the distance between lines. If there are small gaps, the depth doesn't need to be deep, but if there are large gaps, he will carve a depth up to 4mm. This is to make sure the paper doesn't touch the wrong part of the wood and get pigment where it should not.

There is already a `!carve-depth` command [here](https://github.com/natzert/PrintMakingBot/blob/master/commands.yml#L53-L55). I guess I should be changing the both the command and the response?

**Comment by vividkp** [comment]
Yes that sounds right.  We envisioned this one as a synonym, but there's no
reason to have both commands.  !depth is shorter and unambiguous.
Thanks!

**Comment by natzert** [comment]
On that pull request, if you click on the Files Changed tab you will see a green `Review Changes` button on the right side. There are three options there, Comment, Approve, or Request Changes. The Comment and Request Changes options can be done on a line-by-line basis if there are specific things you want changed. If you click the Approve button will allow me to apply the changes.

I can apply the changes because I own this repository but do you want to test approving the Pull Request anyway just to make it easier in the future?

Here is the direct link to the Files Changed tab:

https://github.com/natzert/PrintMakingBot/pull/23/files
